float32[] distance
string front_back
string left_right